### PR TITLE
apply cookiecutter

### LINKED ISSUE #29: use autonumbering for footnote references
see  #27

### PR DESCRIPTION
**Pull request by ahms5**
use cookiecutter to update the doc
Also fixes #29
- Auto-footnotes do not like to be referenced in the first line. 
- It also does not like other strings with square brackets, e.g. [-5, -10] dB.

merges into #26

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.4.0
+current_version = 0.1.0
```
Comment (ahms5) [inline setup.cfg:2]:
```suggestion
current_version = 0.4.0
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -80,18 +81,19 @@ jobs:
 
     steps:
       - checkout
-      - run:
-          name: Install System Dependencies
-          command: sudo apt-get update && sudo apt-get install -y libsndfile1
+   #    - run:
+   #        name: Install System Dependencies
+   #        command: sudo apt-get update && sudo apt-get install -y libsndfile1 texlive-latex-extra dvipng
```
Comment (ahms5) [inline .circleci/config.yml:86]: - run:
          name: Install System Dependencies
          command: sudo apt-get update && sudo apt-get install -y libsndfile1
```suggestion
   #    - run:
   #        name: Install System Dependencies
   #        command: sudo apt-get update && sudo apt-get install -y libsndfile1 texlive-latex-extra dvipng
```

File: .readthedocs.yml [hunk 3]
```diff
@@ -9,8 +9,8 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.10"
-  apt_packages:
-    - libsndfile1
+  # apt_packages:
+  #   - libsndfile1
```
Comment (ahms5) [inline .readthedocs.yml:13]:
```suggestion
  apt_packages:
    - libsndfile1
```

File: README.rst [hunk 4]
```diff
@@ -1,18 +1,32 @@
...
 
+Getting Started
+===============
+
+The `pyfar workshop`_ gives an overview of the most important pyfar
+functionality and is a good starting point. It is part of the
+`pyfar example gallery`_ that also contains more specific and in-depth
+examples that can be executed interactively without a local installation by
+clicking the mybinder.org button on the respective example. The
+`pyfar documentation`_ gives a detailed and complete overview of pyfar. All
```
Comment (mberz) [inline README.rst:27]: This should link to the parato docs

File: README.rst [hunk 5]
```diff
@@ -22,20 +36,18 @@ Use pip to install pyrato
...
 
-Refer to the `contribution guidelines`_ for more information.
+Check out the `contributing guidelines`_ if you want to become part of pyfar.
 
-.. _contribution guidelines: https://github.com/pyfar/pyrato/blob/main/CONTRIBUTING.rst
-.. _examples notebook: https://mybinder.org/v2/gh/pyfar/pyrato/HEAD?filepath=examples%2Fenergy_decay_curves_and_reverberation_time.ipynb
-.. _read the docs: https://pyrato.readthedocs.io/en/stable
+.. _pyfar workshop: https://mybinder.org/v2/gh/pyfar/gallery/main?labpath=docs/gallery/interactive/pyfar_introduction.ipynb
+.. _pyfar example gallery: https://pyfar-gallery.readthedocs.io/en/latest/examples_gallery.html
+.. _pyfar documentation: https://pyfar.readthedocs.io
```
Comment (mberz) [inline README.rst:50]: This should link to the Pyrato docs

**Review by mberz — CHANGES_REQUESTED** [review]

File: pyrato/edc.py [hunk 6]
```diff
@@ -604,11 +602,11 @@ def energy_decay_curve_chu_lundeby(
         channel_independent=False,
         normalize=True,
         plot=False):
-    """ This function combines Chu's and Lundeby's methods:
+    """This function combines Chu's and Lundeby's methods.
 
     The estimated noise level is subtracted before backward integration,
     the impulse response is truncated at the intersection time,
-    and the correction for the truncation is applied [4]_, [5]_, [6]_
+    and the correction for the truncation is applied [#]_, [#]_, [#]_
```
Comment (hoyer-a) [inline pyrato/edc.py:609]:
```suggestion
    and the correction for the truncation is applied [#]_, [#]_, [#]_.
```

**Review by hoyer-a — COMMENTED** [review]
Thanks

**Review by hoyer-a — APPROVED** [review]

**Review by mberz — COMMENTED** [review]
There have been changes in #27, please merge and check if this PR is affected.

**Comment by mberz** [comment]
c30621e03fbd57ebdaa6648a75a2813f08b54c83

**Comment by ahms5** [comment]
done

**Pull request merged**